modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/macros/WorkflowMetricsProcessListView.macro

```diff
@@ -0,0 +1,18 @@
+definition {
+
+	macro assertPaginationComponentPresent {
+		AssertElementPresent(locator1 = "WorkflowMetricsProcessListView#METRICS_WORKFLOW_PAGINATION_PREVIOUS_BUTTON");
+
+		AssertElementPresent(
+			key_pageNumber = "${pageNumber1}",
+			locator1 = "WorkflowMetricsProcessListView#METRICS_WORKFLOW_PAGINATION_NUMBER_BUTTON");
+
+		if (isSet(pageNumber2)) {
+			AssertElementPresent(
+				key_pageNumber = "${pageNumber2}",
+				locator1 = "WorkflowMetricsProcessListView#METRICS_WORKFLOW_PAGINATION_NUMBER_BUTTON");
+		}
```

Comment:
Instead of doing that, you can change pageNumber to be a list and ba called like this:

`WorkflowMetricsProcessListView.assertPaginationComponentPresent(pageNumberList = "1");`

And the code on this macro would be like this:


```suggestion
for (var pageNumber : list "${pageNumberList}") {
    var key_pageNumber = "${pageNumber}";
    
    AssertElementPresent(locator1 = "WorkflowMetricsProcessListView#METRICS_WORKFLOW_PAGINATION_NUMBER_BUTTON");
}
```